rdk/rdk.py

```diff
@@ -4115,7 +4106,7 @@ def __tag_config_rule(self, rule_name, cfn_tags, my_session):
     def __get_lambda_layers(self, my_session, args, params):
         layers = []
         if "SourceRuntime" in params:
-            if params["SourceRuntime"] in ["python3.6-lib", "python3.7-lib", "python3.8-lib"]:
+            if params["SourceRuntime"] in ["python3.7-lib", "python3.8-lib"]:
```

Comment:
```suggestion
            if params["SourceRuntime"] in ["python3.7-lib", "python3.8-lib", "python3.9-lib"]:
```

Reply:
@bmorrissirromb I intentionally wanted to add the missing 3.9 stubs in a subsequent PR and leave this strictly to removal of 3.6.

Is that not desired?

Reply:
Resolved in https://github.com/mbeacom/aws-config-rdk/compare/remove-python36-361...mbeacom:aws-config-rdk:add-python39-362?expand=1

Reply:
Resolved in https://github.com/mbeacom/aws-config-rdk/compare/remove-python36-361...mbeacom:aws-config-rdk:remove-python-copy-364?expand=1 -- will rebase once merged and raise PR.